content/docs/en/elements/components/activity-indicator.md

```diff
@@ -1,5 +1,21 @@
 ---
 title: ActivityIndicator
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_activity_indicator_.activityindicator
 ---
 
-TODO
+Adds a progress indicator that indicates that something is running in the background, prompting the user to pause.
+
+---
+
+```html
+<ActivityIndicator busy="true" />
+```
+
+[> screenshots for=ActivityIndicator <]
+
+
+## Props
+
+| name | type | description |
+|------|------|-------------|
+| `busy` | `Boolean` | if true, it will be active
```

Comment:
should wrap true in backticks so it displays as code:

When `true`, the indicator is active.

Reply:
will remember that.